### ISSUE TITLE
problem fine tuning segment anything model

### ISSUE THREAD

**Issue by sushmanthreddy** [NielsRogge/Transformers-Tutorials#294]
hi, @NielsRogge I have been trying to use fine-tuning segment anything model for the segmentation of cells , have gone through your following tutorial,it is great and have little confusion with bounding box extraction.
if the image has multiple cells and we need to segment it , it is kind of big problem to extract multiple cells bounding box for each cell?does that code mentioned in tutorial can be used??
can we use a prompt or dot representing the cell in an image for training 

for example :
**orginal image**
![image](https://github.com/NielsRogge/Transformers-Tutorials/assets/73489688/d7269adb-23e1-49a8-975f-1051f61aa14a)
**Ground truth**
![image](https://github.com/NielsRogge/Transformers-Tutorials/assets/73489688/d0c76535-795e-4e4c-b2c1-5dbf1179009c)
**position of prompt for sam**A
![image](https://github.com/NielsRogge/Transformers-Tutorials/assets/73489688/467de25d-98bb-40fe-b3c7-f00b164a133b)

can we use this other than the bounding box??
the position of the prompt can be extracted from pixel values as microscopy images are greyscale the pixel value greater than zero position can be extracted 

as mentioned in paper architecture

![image](https://github.com/NielsRogge/Transformers-Tutorials/assets/73489688/7a0fc430-3394-4b0f-a30c-03d2b43f2811)

can we use this method for fine tuning the model?

**Comment by sushmanthreddy** [comment]
@NielsRogge  pls feel free to comment..

**Comment by NielsRogge** [comment]
Hi,

Yes my notebook was just one example of how you could fine-tune SAM, using a bounding box as prompt. You could indeed also fine-tune the model using other types of prompts, like points or rudimentary masks. Technically you could also use text as prompt but the SAM authors didn't release that.

**Comment by karthikdatta98** [comment]
@NielsRogge @sushmanthreddy Can I know how big should the data set be? would 100 examples be enough? also as the pre-trained model was trained on 1024x1024, do I need to stick to that resolution? Looking forward to your reply. Thanks.
